packages/blocks/src/__internal__/utils/gesture.ts

```diff
@@ -158,36 +165,45 @@ export function initMouseEventHandlers(
       e.preventDefault();
     }
 
-    if (!isDragging)
-      onContainerClick(
-        toSelectionEvent(e, getBoundingClientRect, startX, startY)
-      );
-    else
+    if (isDragging) {
       onContainerDragEnd(
         toSelectionEvent(e, getBoundingClientRect, startX, startY, last)
       );
-
-    const horizontalElement = getClosestHorizontalEditor(e.clientY);
-
-    if (!isInsideBlockContainer(e.target) && horizontalElement) {
-      let containerDom = document.querySelector(
-        '.affine-default-page-block-container'
+    } else {
```

Comment:
This `else` also prevents the selection being updated after dragging.
I will upload another video.

Reply:
Dragging out of the frame will cause the selection to be reset.

https://user-images.githubusercontent.com/9910706/215275217-d2edf247-2cca-4744-a7db-e90fddae036a.mp4